apps/pom.xml

```diff
@@ -75,54 +75,54 @@
 		<sonar.jacoco.reportPath>target/jacoco.exec</sonar.jacoco.reportPath>
 		<sonar.jacoco.itReportPath>target/jacoco-it.exec</sonar.jacoco.itReportPath>
 
-		<micrometer.version>1.10.2</micrometer.version>
-		<metrics.version>4.2.15</metrics.version>
+		<micrometer.version>1.10.4</micrometer.version>
+		<metrics.version>4.2.16</metrics.version>
 		<dropwizard-metrics-newrelic.version>0.8.0</dropwizard-metrics-newrelic.version>
 		<telemetry-http-okhttp.version>0.15.0</telemetry-http-okhttp.version>
-		<aws-java-sdk-bom.version>1.12.375</aws-java-sdk-bom.version>
+		<aws-java-sdk-bom.version>1.12.410</aws-java-sdk-bom.version>
 		<spring-data-jpa.version>2.7.5</spring-data-jpa.version>
 		<spring-framework.version>5.3.23</spring-framework.version>
 		<!-- Newer (6.0) release versions for hibernate are available for some hibernate components
 		but not all (in particular, entity-manager does not have a 6.0 version available yet).
 		Staying on latest 5.x version until all components have 6.0 versions available. -->
 		<hibernate.version>5.6.10.Final</hibernate.version>
-		<junit.version>5.9.1</junit.version>
+		<junit.version>5.9.2</junit.version>
 		<javadoc.version>3.4.1</javadoc.version>
 
 		<!-- Jackson versions should be kept in sync when possible but can diverge slightly
 			when vulnerabilities or other fixes affect some but not all components -->
-		<jackson-databind.version>2.14.1</jackson-databind.version>
-		<jackson-datatype-jdk8.version>2.14.1</jackson-datatype-jdk8.version>
-		<jackson-dataformat-yaml.version>2.14.1</jackson-dataformat-yaml.version>
-		<jackson-datatype-jsr310.version>2.14.1</jackson-datatype-jsr310.version>
+		<jackson-databind.version>2.14.2</jackson-databind.version>
+		<jackson-datatype-jdk8.version>2.14.2</jackson-datatype-jdk8.version>
+		<jackson-dataformat-yaml.version>2.14.2</jackson-dataformat-yaml.version>
+		<jackson-datatype-jsr310.version>2.14.2</jackson-datatype-jsr310.version>
 
-		<mockito.version>4.11.0</mockito.version>
-		<postgresql.version>42.5.1</postgresql.version>
+		<mockito.version>5.1.1</mockito.version>
```

Comment:
Big deal; mockito 5 allows us to mock static and final classes without a bunch of workaround now!